### PR TITLE
Split redo between jdebp and apenwarr versions

### LINKED ISSUE #189: Split redo
There are currently 2 versions of [redo][] in repology which are conflated. One is by [jdebp][], which currently has version 1.4, and the other by [apenwarr][], which has version 0.41.

Could these be split?

[redo]:https://repology.org/project/redo/versions
[jdebp]:https://jdebp.eu/Softwares/redo/
[apenwarr]:https://github.com/apenwarr/redo

### PR DESCRIPTION
**Pull request by Ikke**
There are multiple implementations of redo. Repology currently shows 2 of them under the same name. One by jdebp and one by apenwarr. Split them into redo-jdebp and redo-apenwarr.

Fixes #189

### CODE REVIEW

File: 850.split-ambiguities/r.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@
 - { name: rdd, wwwpart: r043v, setname: rdd-redis }
 - { name: rdd, warning: "please classify me" }
 
+- { name: redo, wwwpart: jdebp.eu, setname: redo-jdebp }
+- { name: redo, wwwpart: [redo.readthedocs.io, github.com/apenwarr], setname: redo-apenwarr }
```
Comment (AMDmi3) [inline 850.split-ambiguities/r.yaml]: Looks like this needs to be extended with `apenwarr.ca`

File: 850.split-ambiguities/r.yaml [hunk 2]
```diff
@@ -12,6 +12,9 @@
...
+
```
Comment (AMDmi3) [inline 850.split-ambiguities/r.yaml]: - Needs additional rules for repositories which lack homepage information, namely exherbo and solus.
- Also needs `- { name: redo, warning: "please classify me" }` to get notified when new packages which need split appear

Comment (Ikke) [reply 850.split-ambiguities/r.yaml]: Any suggestion what I could use in the case of exherbo and solus? I'm not too familiar with the repology data structure.

Comment (AMDmi3) [reply 850.split-ambiguities/r.yaml]: https://github.com/repology/repology-rules#ruleset

Comment (Ikke) [reply 850.split-ambiguities/r.yaml]: Yes, thanks I found that. Is there any other place than https://repology.org/project/redo/information to see the actual metadata that is available for each distro/repo</

Comment (AMDmi3) [reply 850.split-ambiguities/r.yaml]: https://repology.org/project/redo/packages has enough information for adding new rules

Comment (Ikke) [reply 850.split-ambiguities/r.yaml]: Updated the PR to include solus and exherbo.

**Pull request merged**